Makefile.common

```diff
@@ -142,12 +158,41 @@ SOURCES_C += \
 	$(CORE_DIR)/subprojects/md5/md5.c
 COREFLAGS += -I$(CORE_DIR)/subprojects/md5
 
+# libpng
+ifeq ($(SYSTEM_LIBPNG), 1)
```

Comment:
The last change i'd like before merge is adding each SYSTEM_ var to `Makefile` at the top, seperately spaced by `\n` with `?= 0`

Reply:
Purely so i keep track of them :P

Reply:
Sure thing!